apps/src/StudioApp.js

```diff
@@ -1212,7 +1214,7 @@ StudioApp.prototype.showInstructionsDialog_ = function (level, autoClose, showHi
 
   var hideFn = _.bind(function () {
     // Momentarily flash the instruction block white then back to regular.
-    if ($(endTargetSelector).length) {
+    if ($(endTargetSelector).length && !instructionsInTopPane) {
```

Comment:
Second nit: `$(endTargetSelector)` happens twice if you do enter the conditional.

I understand that to satisfy both nits you'd have to do an assignment inside the condition, or nested conditions, neither being a great solution.  Feel free to pick one or the other of these to fix and go with it.